docs/audit_report/src/00_09_introduction.rst

```diff
@@ -2,7 +2,7 @@ Introduction
 ============
 
 This audit report summarizes the review results of changes to the Botan library code
-base between the tagged releases 3.0.0-alpha1 and 3.1.1.
+base between the tagged releases 3.1.1 and 3.2.0.
```

Comment:
Maybe use substitutions for base and target versions, so it only needs to be changed at one place?

Reply:
Yeah, that'd be great. I'll add it to the TODO list.